eng/Baseline.xml

```diff
@@ -1,4 +1,4 @@
-<Baseline Version="2.2.0">
+<Baseline Version="2.2.2">
```

Comment:
```suggestion
<Baseline Version="2.2.1">
```

Reply:
This should be the most recent released version, not the currently building version.